tests/trainer/connectors/test_checkpoint_connector.py

```diff
@@ -126,3 +131,56 @@ def test_hpc_max_ckpt_version(tmpdir):
     assert trainer.checkpoint_connector.hpc_resume_path == str(tmpdir / "hpc_ckpt_33.ckpt")
     assert trainer.checkpoint_connector.max_ckpt_version_in_folder(tmpdir) == 33
     assert trainer.checkpoint_connector.max_ckpt_version_in_folder(tmpdir / "not" / "existing") is None
+
+
+@mock.patch.dict(os.environ, {"PL_FAULT_TOLERANT_TRAINING": "1"})
+@RunIf(min_torch="1.7.0")
+def test_loops_restore(tmpdir):
```

Comment:
This coudl be a simple unit-test where `restore_loops` is called and the different Loops are mocked to check that their respective `load_state_dict` functions were called.

Reply:
are we checking anywhere else whether loop's state_dict is restored correctly or not?? couldn't find it that's why added these assertions to check everything is working as expected.

Reply:
Yes, all the tests in `tests/loops/test_loops.py` do

Reply:
i see.. will update